pkg/ui/console/client.go

```diff
@@ -660,7 +669,7 @@ func printPolicyResponse(results []*policy.ExecutionResult) {
 func createOutputTable(res *policy.QueryResult) string {
 	data := make([][]string, 0)
 	for rowIndex := range res.Data {
-		rowData := []string{}
+		rowData := make([]string, 1)
```

Comment:
We're doing an `append` below, so this would mean rowData[0] is always empty. FYI, probably wrong. Did you mean `make([]string, 0, 1)`?